src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java

```diff
@@ -790,17 +790,17 @@ private static String toStringConvertForArrayValue(Object value) {
                            })
                            .collect(Collectors.joining(","));
         }
-        else if (value instanceof int[]) {
-            result = Arrays.toString((int[]) value).replaceAll("[\\[\\]\\s]", "");
```

Comment:
its not just about bumping the `java.version` field to (constantly) complete a migration.